Block/Product/YotpoReviewRenderer.php

```diff
@@ -18,12 +18,26 @@ public function getReviewsSummaryHtml(
         $displayIfNoReviews = false
     ) {
         
+        $objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+        $storeManager = $objectManager->get('\Magento\Store\Model\StoreManagerInterface');
+        $cp = $objectManager->get('\Magento\Framework\App\Request\Http');
         $enableBottomlineCategoryPage = $this->isBottomlineCategoryEnabled();
-	$enableMagentoDefaultReviews = $this->isMagentoDefaultReviewsEnabled();
+		$enableMagentoDefaultReviews = $this->isMagentoDefaultReviewsEnabled();
+		
         if ($enableBottomlineCategoryPage) {
-            return $this->showCategoryBottomLine($product);
+			if ($cp->getFullActionName()==='cms_index_index'){
+				return $this->showCategoryBottomLine($product);
+			}
+			if ($cp->getFullActionName()==='catalog_category_view'){
```

Comment:
consider merging the 2 if statements to one with "or" because the body is the same in both statements